src/Contracts/Http/ResponsesInterface.php

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-use \Neomerx\JsonApi\Document\Error;
+use \Neomerx\JsonApi\Contracts\Document\ErrorInterface;
```

Comment:
Can you please move this line down?

use \aaaaaa;
use \bbbbbbbb;
use \ccccccccccccc;